app/boards/shields/leeloo/leeloo.conf

```diff
@@ -1,9 +1,16 @@
 # Copyright (c) 2022 The ZMK Contributors
 # SPDX-License-Identifier: MIT
 
-# Uncomment the following line to enable the OLED Display
+# Uncomment the following line to enable the OLED Display or nice!view Display
 # CONFIG_ZMK_DISPLAY=y
 
+# Uncomment to turn off WPM Status.
+# CONFIG_ZMK_WIDGET_WPM_STATUS=n
+
+# Uncomment to invert colour, if using nice!view Displays
+# CONFIG_LVGL_USE_THEME_MONO=y
```

Comment:
I don't think this is true anymore, you might want to test `CONFIG_ZMK_DISPLAY_INVERT=y` instead: https://zmk.dev/docs/config/displays

(Also applies to the micro)

Reply:
Thank you @caksoylar!  You are right, this is an outdated config setting.